src/components/Toolbar/__tests__/Toolbar.test.jsx

```diff
@@ -6,15 +6,21 @@ import { useUser } from '../../../contexts/UserContext';
 jest.mock('../../../contexts/UserContext');
 const { UserProvider } = jest.requireActual('../../../contexts/UserContext');
 
-describe('Hamburger Toolbar Component Tests', function() {
-  test('should show navigation links', () => {
-    useUser.mockReturnValue([null, null]);
-    const { queryAllByRole, queryByText } = renderWithRouter(
+describe('Toolbar Component Tests', function() {
+  function setup(useUserValue = [null, null]) {
+    useUser.mockReturnValue(useUserValue);
+    const { queryAllByRole, queryByText, queryByTestId, debug } = renderWithRouter(
```

Comment:
Also i don't think queryByTestId and debug are ever used in these tests, if not I'd remove